guides/source/getting_started.textile

```diff
@@ -118,7 +118,7 @@ Just add it to your apps Gemfile
 
 <ruby>
 # Gemfile
-gem 'alchemy-devise', github: 'magiclabs/alchemy-devise', branch: '2.0-stable'
+gem 'alchemy-devise', github: 'magiclabs/alchemy-devise', branch: '3.4-stable'
```

Comment:
Could you please also update the repo URL to 

`AlchemyCMS/alchemy-devise`

Thanks